mmseg/datasets/pipelines/test_time_aug.py

```diff
@@ -58,20 +58,27 @@ def __init__(self,
                  flip_direction='horizontal'):
         self.transforms = Compose(transforms)
         if img_ratios is not None:
-            # mode 1: given a scale and a range of image ratio
             img_ratios = img_ratios if isinstance(img_ratios,
                                                   list) else [img_ratios]
             assert mmcv.is_list_of(img_ratios, float)
-            assert isinstance(img_scale, tuple) and len(img_scale) == 2
+        if img_scale is None:
+            # mode 1: given img_scale=None and a range of image ratio
+            self.img_scale = None
+            assert mmcv.is_list_of(img_ratios, float)
+        elif isinstance(img_scale,
+                        tuple) and len(img_scale) == 2 and mmcv.is_list_of(
```

Comment:
We move move `len(img_scale)` condition to assert.

Reply:
Done!

Reply:
`elif isinstance(img_scale, tuple) and mmcv.is_list_of(img_ratios, float):
assert len(img_scale) == 2 `

Reply:
Fixed in the latest commit and please check it again.